src/ui/map.js

```diff
@@ -789,12 +789,17 @@ class Map extends Camera {
      * Returns the map's geographical bounds. When the bearing or pitch is non-zero, the visible region is not
      * an axis-aligned rectangle, and the result is the smallest bounds that encompasses the visible region.
      * If a padding is set on the map, the bounds returned are for the inset.
+     * This API isn't supported with globe projection.
      *
      * @returns {LngLatBounds} The geographical bounds of the map as {@link LngLatBounds}.
      * @example
      * const bounds = map.getBounds();
      */
-    getBounds(): LngLatBounds {
+    getBounds(): LngLatBounds | null {
```

Comment:
Returning `null` for a function that previously always returned a value seems like it will likely cause an error for some users upgrading to globe.

But as we discussed, users can check for the `null` return and this option is better than returning an inaccurate bounds.